test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -26,20 +26,19 @@ class TestFunctional(common_utils.TorchaudioTestCase):
 
     backend = "default"
 
-    def assert_batch_consistency(self, functional, batch, *args, atol=1e-8, rtol=1e-5, seed=42, **kwargs):
-        n = batch.size(0)
+    def assert_batch_consistency(self, functional, inputs, *args, atol=1e-8, rtol=1e-5, seed=42, **kwargs):
```

Comment:
With this number of arguments and the combination of args and kwargs, it's error-prone. I think it's better to use keyword-only.

```suggestion
    def assert_batch_consistency(self, functional, *, inputs, args=None, kwargs=None, atol=1e-8, rtol=1e-5, seed=42):
```



```suggestion
    def assert_batch_consistency(self, functional, *inputs, *, atol=1e-8, rtol=1e-5, seed=42, args=None, kwargs=None):
```

Reply:
If not using ``kwargs``, how to pass the keywords to `functional`?